src/nvim/eval/funcs.h

```diff
@@ -4,8 +4,6 @@
 #include "nvim/buffer_defs.h"
 #include "nvim/eval/typval.h"
 
-typedef void (*FunPtr)(void);
-
```

Comment:
I don't think this should be moved.

Reply:
It is required in `typval.h` and `funcs.h` includes that. How else could we organise this?